erpnext/patches/v9_0/set_pos_profile_name.py

```diff
@@ -11,7 +11,7 @@ def execute():
 	for pos in frappe.get_all(doctype, filters={'disabled': 0}):
 		doc = frappe.get_doc(doctype, pos.name)
 
-		if not doc.user and doc.pos_profile_name: continue
+		if not doc.user or doc.pos_profile_name: continue
```

Comment:
So what will be the profile-name if user is not there? Maybe we can call it "Standard"